src/app/series/directives/series-feed.component.ts

```diff
@@ -66,57 +69,84 @@ import { SeriesModel, StoryModel } from '../../shared';
       </div>
 
     </section>
-    </div>
+    <prx-spinner *ngIf="!isLoaded" [class.paging]="isPaging"></prx-spinner>
+  </div>
   `,
   styleUrls: ['./series-feed.component.css']
 })
 
-export class SeriesFeedComponent implements OnDestroy {
+export class SeriesFeedComponent implements OnInit, OnDestroy {
 
   isLoaded = false;
-  noStories: boolean;
-  series: SeriesModel;
-  publicStories: StoryModel[] = [];
-  futurePublicStories: StoryModel[] = [];
-  privateStories: StoryModel[] = [];
+  isPaging = false;
+  noStories = false;
+  series: HalDoc;
+  publicStories: HalDoc[] = [];
```

Comment:
Simplified from the Models down to regular HalDocs (which tend to work better/faster when things are read-only).